docs/book/src/tasks/healthcheck.md

```diff
@@ -38,7 +38,9 @@ spec:
   # (Optional) maxUnhealthy prevents further remediation if the cluster is already partially unhealthy
   maxUnhealthy: 40%
   # (Optional) nodeStartupTimeout determines how long a MachineHealthCheck should wait for
-  # a Node to join the cluster, before considering a Machine unhealthy
+  # a Node to join the cluster, before considering a Machine unhealthy.
+  # Defaults to 10 minutes if not specified.
+  # Set to 0 to disable this check.
```

Comment:
Can we be a bit more precise on what "disable this check" means? Are we talking about the entire MHC? the node joined check? What's the behavior the user should expect when this is set to 0?

Reply:
Good catch, I've tried to expand this to make it more obvious, now wondering if this is too long for a comment on the field, perhaps it would be better to move to a separate heading which details the node startup timeout behaviour more fully?